Integrations/MailListener-POP3/MailListener-POP3.py

```diff
@@ -0,0 +1,384 @@
+import demistomock as demisto
+
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import poplib
+import base64
+import quopri
+from email.parser import Parser
+from htmlentitydefs import name2codepoint
+from HTMLParser import HTMLParser, HTMLParseError
+
+
+''' GLOBALS/PARAMS '''
+SERVER = demisto.params().get('server', '')
+EMAIL = demisto.params().get('email', '')
+PASSWORD = demisto.params().get('password', '')
+PORT = int(demisto.params().get('port', '995'))
+SSL = demisto.params().get('ssl')
+FETCH_TIME = demisto.params().get('fetch_time', '7 days')
+
+# pop3 server connection object.
+pop3_server_conn = None  # type: ignore
+
+TIME_REGEX = re.compile(r'^([\w,\d: ]*) (([+-]{1})(\d{2}):?(\d{2}))?[\s\w\(\)]*$')
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+def connect_pop3_server():
+    global pop3_server_conn
+
+    try:
+        if pop3_server_conn is None:
+            if SSL:
+                pop3_server_conn = poplib.POP3_SSL(SERVER, PORT)  # type: ignore
+            else:
+                pop3_server_conn = poplib.POP3(SERVER, PORT)  # type: ignore
+
+            pop3_server_conn.getwelcome()  # type: ignore
+            pop3_server_conn.user(EMAIL)  # type: ignore
+            pop3_server_conn.pass_(PASSWORD)  # type: ignore
+    except Exception as e:
+        raise e
+
+
+def close_pop3_server_connection():
+    global pop3_server_conn
+    if pop3_server_conn is not None:
+        pop3_server_conn.quit()
+        pop3_server_conn = None
+
+
+def get_user_emails():
+    connect_pop3_server()
+    (resp_message, mails_list, octets) = pop3_server_conn.list()  # type: ignore
```

Comment:
if you are not using a returned value, don't store it.
```suggestion
    _, mails_list, _ = pop3_server_conn.list()  # type: ignore
```